backend/src/openbeheer/types/_open_beheer.py

```diff
@@ -558,31 +564,36 @@ def _resultaat_as_option(resultaat: LAXResultaat, **kwargs) -> OBOption[Resultaa
     )
     label = " - ".join(label_values)
     return OBOption(
-        label=f"{label} - {resultaat.proces_type}",
+        label=f"{label}",
         value=resultaat.url,
     )
 
 
+def fetch_resultaat_options(
```

Comment:
Should we include selectielijst_resultaat in the name to be very explicit that these are not OZ resultaten?